geomesa-tools/examples/ingest/csv/example1.conf

```diff
@@ -22,7 +22,7 @@
       {name = "id", transform = "$1::int"},
       {name = "name", transform = "$2::string"},
       {name = "age", transform = "$3::int"},
-      {name = "lastseen", transform = "$4::date"},
```

Comment:
for consistency, it would be nice if all the sample files were something like:
example.csv
example-csv.conf
etc